File: src/main/java/namingservice/requests/RegistrationRequest.java
package namingservice.requests;

import com.google.gson.Gson;
import requestreplyapi.Registry.ExtendedEntry;

public class RegistrationRequest extends Request{
    private ExtendedEntry entry_data;
    private String entry_name;

    public RegistrationRequest(String entry_name, ExtendedEntry entry_data) {
        super("registration_request");
        this.entry_data = entry_data;
        this.entry_name = entry_name;
    }

    public ExtendedEntry getEntry_data() {
        return entry_data;
    }

    public void setEntry_data(ExtendedEntry entry_data) {
        this.entry_data = entry_data;
    }

    public String getEntry_name() {
        return entry_name;
    }

    public void setEntry_name(String entry_name) {
        this.entry_name = entry_name;
    }


}

File: ObjectRequestBrokerCS/ObjectRequestBrokerCS/namingservice/requests/RegistrationRequest.cs
namespace namingservice.requests
{
	using ExtendedEntry = requestreplyapi.entries.ExtendedEntry;

	public class RegistrationRequest : Request
	{
		private ExtendedEntry entry_data;
		private string entry_name;

		public RegistrationRequest(string entry_name, ExtendedEntry entry_data) : base("registration_request")
		{
			this.entry_data = entry_data;
			this.entry_name = entry_name;
		}

		public RegistrationRequest()
		{
		}

		public virtual ExtendedEntry Entry_data
		{
			get
			{
				return entry_data;
			}
			set
			{
				this.entry_data = value;
			}
		}


		public virtual string Entry_name
		{
			get
			{
				return entry_name;
			}
			set
			{
				this.entry_name = value;
			}
		}



	}

}